interoperability_report.py

```diff
@@ -447,10 +447,10 @@ def run_test(
             entity_type.append(f'Publisher_{publisher_number}')
 
         elif('-S ' in parameters[i] or parameters[i].endswith('-S')):
-            # Wait 1 second before running the subscriber if durability is
-            # provided
-            if '-D ' in parameters[i]:
-                time.sleep(1)
+            # Wait 1 second before running the subscriber to avoid conflicts between
+            # the programs on startup
+            time.sleep(1)
```

Comment:
I'm not sure about the consequences of this change for non-durability tests. I think it should be safe, but I'd like to test this change beforehand, can you retarget the PR to a different branch? So we can run the tests in that branch before merging this change to master

Reply:
I removed this change from the pull request, so now it only adds the InterCOM DDS stuff. I will create a new PR with the sleep.